src/interactions/panZoomInteraction.ts

```diff
@@ -1,4 +1,8 @@
 namespace Plottable.Interactions {
+
+  export type PanCallback = (e: Event) => void;
+  export type ZoomCallback = (e: Event) => void;
```

Comment:
Yeah, I was a little unsure as to what to return, but you can take multiple
scales right, so would it be an array of domains returned?

On Thu, 14 Jul 2016, 19:27 jtlan, <notifications@github.com> wrote:

> In src/interactions/panZoomInteraction.ts
> <https://github.com/palantir/plottable/pull/3087#discussion_r70857970>:
>
> > @@ -1,4 +1,8 @@
> >  namespace Plottable.Interactions {
> > +
> > +  export type PanCallback = (e: Event) => void;
> > +  export type ZoomCallback = (e: Event) => void;
>
> Not sure these callbacks should take Events -- that's sort of atypical if
> you compare to the callbacks passed to Click, Drag, and Pointer
> Interactions. I would expect the final domains of the scales or something
> like that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/plottable/pull/3087/files/cf951aba815a4c8d2526d68def596f8869c1ed41#r70857970>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AElqXol0-UeMq4VU9_qldkW-AYz-FEfmks5qVn-bgaJpZM4JMMcF>
> .
>